### PR TITLE
[ADD] Module ir_action_mail_translate

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: ir_action_mail_translate/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    """,
+    'depends': [
+        'base',
+    ],
+    'data': [
+    ],
+    'demo': [],
+    'test': [],
+    'external_dependencies': {
+        'python': ['num2words'],
```
Comment (max3903) [inline ir_action_mail_translate/__openerp__.py]: Do you really need num2words ?

Comment (ghost) [inline ir_action_mail_translate/__openerp__.py]: No, it's a mistake

**Comment by ghost** [comment]
@max3903 You may remove needs fixing label.

File: ir_action_mail_translate/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    'author': 'Savoir-faire Linux',
+    'website': 'http://www.savoirfairelinux.com',
+    'category': 'other',
+    'license': 'AGPL-3',
+    'description': """
+Server Action Email Translate
+=============================
+
+This module allows to translate server action email messages in the
+recipient's language.
```
Comment (max3903) [inline ir_action_mail_translate/__openerp__.py]: Can you change the description to:

```
By default, emails used in server action are sent in english, even if they are translated. 
This module allows to send those email messages in the partner's language or 
the admin user language if there is no partner yet (HR applicant for example).
```

File: ir_action_mail_translate/models/ir_actions_server.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                'pool': self.pool,
+                'time': time,
+                'cr': cr,
+                'context': dict(context),
+                'uid': uid,
+                'user': recipient,
+            }
+
+            try:
+                address = eval(str(action.email), cxt)
```
Comment (StefanRijnhart) [inline ir_action_mail_translate/models/ir_actions_server.py]: I think you should use safe_eval from openerp.tools.safe_eval.
Do you need the string casting?

File: ir_action_mail_translate/models/ir_actions_server.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            except Exception:
```
Comment (StefanRijnhart) [inline ir_action_mail_translate/models/ir_actions_server.py:65]: Can you catch a more specific exception? -- edit -- void as per the above.

**Comment by StefanRijnhart** [comment]
Looks good, apart from the little nag!

File: ir_action_mail_translate/models/ir_actions_server.py (hunk 3 above)
Comment (ghost) [inline ir_action_mail_translate/models/ir_actions_server.py]: @StefanRijnhart I admit this part of the code is a bit odd. I copied the 4 lines from addons/base/ir/ir_actions.py line 624. I would keep it this way to ensure that in every case, the value of the variable address will be the same here as in addons/base/ir/ir_actions.py. What do you think ?

Comment (StefanRijnhart) [inline ir_action_mail_translate/models/ir_actions_server.py]: OK. Just use safe_eval, because that is what ir_actions.py uses (see imports in that file).

Comment (ghost) [inline ir_action_mail_translate/models/ir_actions_server.py]: Done

**Comment by ghost** [comment]
@max3903 you may remove needs fixing label

**Comment by StefanRijnhart** [comment]
Thanks! :+1:

**Comment by dreispt** [comment]
Isn't this also a candidate for the fresh `OCA/social` repo?

**Comment by ghost** [comment]
@StefanRijnhart @max3903 what do you think ? should I move the PR to OCA/social ? I think it would be more acurate than server-tools

**Comment by StefanRijnhart** [comment]
Go ahead if you like!

**Comment by ghost** [comment]
@dreispt Could you create the branch 7.0 in OCA/social please ?

**Pull request closed without merging**